packages/cdktf/lib/backends/cloud-backend.ts

```diff
@@ -40,6 +40,17 @@ export class CloudBackend extends TerraformBackend {
     };
   }
 
+  public toMetadata() {
+    const cloud = [
```

Comment:
Nit: Can we dedupe this into a function?

Reply:
we just might 😁
Are you fine with putting this into a function in the cloud backend and reusing it in the old remote backend?

Reply:
yeah seems like a good idea